contrib/node/tests/python/pants_test/contrib/node/tasks/test_node_bundle_integration.py

```diff
@@ -92,6 +95,28 @@ def test_bundle_jvm_binary_with_node_module(self):
       # Transitive dependency that marked as not generating artifacts should not be included.
       self.assertTrue('web-build-tool' not in os.listdir(temp_dir))
 
+  def test_bundle_jvm_binary_with_node_module_invalidation(self):
```

Comment:
Should it be called `with_node_module_invalidation` since it's not the node module that is changing?